packages/dkms.conf

```diff
@@ -0,0 +1,5 @@
+PACKAGE_NAME="gdrdrv"
+PACKAGE_VERSION="@VERSION@"
+BUILT_MODULE_NAME[0]="gdrdrv"
+DEST_MODULE_LOCATION[0]="/kernel/drivers/video/fbdev/nvidia/"
```

Comment:
That does not look like a good location. 
What is the standard for add-on modules ?

Reply:
From dkms manpage:

```
DEST_MODULE_LOCATION[#]=
              This directive specifies the destination where a module  should  be  installed  to,
              once  compiled.   It also is used for finding original_modules.  This is a required
              directive, except as noted below. This directive must start with the text "/kernel"
              which  is  in reference to /lib/modules/<kernelversion>/kernel.  Note that for each
              module within a dkms package, the numeric value of # must be the same for  each  of
              BUILT_MODULE_NAME, BUILT_MODULE_LOCATION, DEST_MODULE_NAME and DEST_MODULE_LOCATION
              and     that     the      numbering      should      start      at      0      (eg.
              DEST_MODULE_LOCATION[0]="/kernel/drivers/something/"
              DEST_MODULE_LOCATION[1]="/kernel/drivers/other/").

              DEST_MODULE_LOCATION is ignored on Fedora Core 6 and  higher,  Red  Hat  Enterprise
              Linux  5 and higher, Novell SuSE Linux Enterprise Server 10 and higher, Novell SuSE
              Linux 10.0 and  higher,  and  Ubuntu.  Instead,  the  proper  distribution-specific
              directory is used.
```

NVIDIA driver installs the modules in `/lib/modules/<kver>/kernel/drivers/video/`.
mlx5*.ko are found in two locations:
- `/lib/modules/<kver>/kernel/drivers/infiniband/hw/mlx5/`.
- `/lib/modules/<kver>/kernel/drivers/net/ethernet/mellanox/mlx5/core/`

An example in https://wiki.archlinux.org/index.php/Dynamic_Kernel_Module_Support suggests `/lib/modules/<kver>/kernel/drivers/misc`.

An example in https://wiki.debian.org/KernelDKMS uses `/lib/modules/<kver>/extra/`.

Those examples do not even follow the dkms manpage. 

I do not know the best practice, but technically I think that anywhere inside `/lib/modules/<kver>/` is fine. However, our rpm and deb packages should install gdrdrv.ko in the same location -- just realize that they are different now.

Any suggestion? Should we put gdrdrv.ko in the same location as nvidia*.ko or in a different location?